.gitignore

```diff
@@ -44,6 +45,7 @@ Carthage/Build
 fastlane/Preview.html
 fastlane/report.xml
 fastlane/README.md
+fastlane/screenshots/
```

Comment:
Rather than removing the `.gitignore` change, I went ahead and added the lane we'll use later on in CI, and configured it with the "proper" folder.